packages/sanity/src/core/bundles/components/dialog/BundleDetailsDialog.tsx

```diff
@@ -67,8 +60,9 @@ export function BundleDetailsDialog(props: BundleDetailsDialogProps): JSX.Elemen
         try {
           event.preventDefault()
           setIsSubmitting(true)
-          await bundleOperation(value)
-          setValue(value)
+
+          const submitValue = {...value, title: value.title?.trim()}
```

Comment:
🤏 nitpick, maybe move the value / title trim() to its own const since it's being used in multiple places? :)